tests/zkevm/test_worst_compute.py

```diff
@@ -955,3 +966,99 @@ def test_empty_block(
         post={},
         blocks=[Block(txs=[])],
     )
+
+
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.slow()
+def test_amortized_bn128_pairings(
```

Comment:
This is the new test that optimizes for number of miller loops. That is, amortize the base cost (final exponentiation) in as many miller loops as possible while keeping in check quadratic memory expansion costs.